### PR TITLE
Fix part of #3474: Merged faq single activity into one layout file and updated dimens.xml (land),(sw600dp-land)

### LINKED ISSUE #3474: Convert faq_single_activity into single layout file
Currently there are 3 versions of `faq_single_activity .xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in the `dimens.xml` file and use it accordingly.

Note: In PR, make sure you add before and after screenshots of `mobile-portrait`, `mobile-landscape`, `tablet-portrait`, and `tablet-landscape` for comparison and make sure that there is no difference between before and after UI.

### PR DESCRIPTION
**Pull request by rishidyno**
Fix #3474

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rishidyno** [comment]
Fix #3474

I have merged the faq_single_activity.xml file into a single file. I have deleted unnecessary values in dimens.xml.

Screenshots before merging into a single file.

mobile-portrait
mobile-landscape
tablet-portrait
tablet-landscape

Screenshots after merging into a single file.
mobile-portrait
mobile-landscape
tablet-landscape
tablet-portrait

Earlier it crashed when in the tablet landscape mode but not the dimension does not match as before because of 
android:layout_width="match_parent"
which was 560dp in the original version of the landscape

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (prayutsu) [inline .idea/misc.xml]: These changes are unwanted, revert them.

File: .idea/runConfigurations.xml [hunk 2]
```diff
@@ -3,10 +3,11 @@
   <component name="RunConfigurationProducerService">
     <option name="ignoredProducers">
       <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.AllInPackageGradleConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestClassGradleConfigurationProducer" />
         <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestMethodGradleConfigurationProducer" />
       </set>
```
Comment (prayutsu) [inline .idea/runConfigurations.xml]: Ditto

**Review by prayutsu — CHANGES_REQUESTED** [review]
@rishidyno Please follow the PR checklist and link the related issue with the PR.
Also please revert the unwanted changes.

**Comment by rishidyno** [comment]
@prayutsu I have done some changes to the code but am still not able to resolve the issue but working on it.

**Comment by rishidyno** [comment]
This PR had some issues so I have closed it and I have created a new PR -> https://github.com/oppia/oppia-android/pull/3526

**Pull request closed without merging**